pkg/cmd/repo/create/create.go

```diff
@@ -105,6 +106,7 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 	cmd.Flags().BoolVar(&opts.Private, "private", false, "Make the new repository private")
 	cmd.Flags().BoolVar(&opts.Internal, "internal", false, "Make the new repository internal")
 	cmd.Flags().BoolVarP(&opts.ConfirmSubmit, "confirm", "y", false, "Confirm the submission directly")
+	cmd.Flags().StringVarP(&opts.BaseDir, "base-dir", "B", "", "The base directory that will be used for repo initialization after creation.")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&opts.BaseDir, "base-dir", "B", "", "The base directory that will be used for repo initialization after creation")
```

Pardon my intrusion here, I’m just exploring the issues/PRs for this repo 😅 This is quite pedantic, but could the period be removed here to be consistent with the help text for the other flags? (None of them have periods.)

Reply:
That makes sense. Updated.